src/test/java/be/seeseemelk/mockbukkit/ServerMockTest.java

```diff
@@ -544,6 +543,72 @@ public void testDefaultPotionEffects()
 		}
 	}
 
+	@Test
+	public void testYamlSerialization()
+	{
+		final String SERVER_SETTING_FILE = "bukkit.yml";
+		File file = null;
+		try {
+			Path tempFile = Files.createTempFile(SERVER_SETTING_FILE, String.valueOf(Math.random()));
+			file = new File(tempFile.toUri());
+			file.deleteOnExit();
+		}
+		catch (Exception e)
+		{
+			fail(e.getMessage());
+		}
```

Comment:
You don't need a try-catch here. You can simply define the test method with `throws Exception`. JUnit will detect an exception as a failed test, and it will pretty print the exception. It also makes the test much cleaner.